src/vm/interpreter.cpp

```diff
@@ -11817,7 +11817,7 @@ void Interpreter::PrintValue(InterpreterType it, BYTE* valAddr)
                 {
                     fprintf(GetLogFile(), " ");
                 }
-                fprintf(GetLogFile(), "0x%p", valAddr[i]);
+                fprintf(GetLogFile(), "0x%b", valAddr[i]);
```

Comment:
I agree that the original `%p` is not right, but `%b` does not seem right either - it is not [documented printf string type] (https://msdn.microsoft.com/en-us/library/hf4y5e3w.aspx). I think it should be `%02x`.